deeppavlov/models/preprocessors/capitalization.py

```diff
@@ -108,8 +118,8 @@ def process_word(word: str, to_lower: bool = False,
     return tuple(answer)
 
 
-@register('lowercase_preprocessor')
-class LowercasePreprocessor(Component):
+@register('char_splitting_lowercase_preprocessor')
+class CharSplittingLowercasePreprocessor(Component):
```

Comment:
The name is really long. Can we come up with something better?

Reply:
How many words is ok? 
PS. It is a holy war about short names vs. precise names

Reply:
Here is a problem that we have a fusion of operations 
- splitting into characters and 
- lowercasing data with decoration of sequence by capitalization prefix (or optionally suffix).

Reply:
Check you changes once more for typos and grammar: THE foLLowing operations